lighthouse-core/report/html/report-styles.css

```diff
@@ -66,7 +67,7 @@
   --color-informative: var(--color-blue-900);
   --color-pass-secondary: var(--color-green-700);
   --color-pass: var(--color-green);
-  --color-hover: var(--color-gray-50);
+  --color-hover: var(--color-gray-50);  
```

Comment:
could you alphabetize this? you can just select all the lines and "sort" in your IDE–we keep everything here sorted.

also I think there is some whitespace at the end of this line (and a couple others)

Reply:
Corrected these